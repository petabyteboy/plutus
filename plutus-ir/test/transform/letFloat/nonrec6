[(lam x (all a (type) (fun a a))
(let (nonrec)
     (termbind (strict) (vardecl i (con integer))
      (let (nonrec)
           (termbind (strict) (vardecl j (con integer))
            [(builtin addInteger) (con 1) x])
            j)
       )
     i
 ))
(con 1)]
