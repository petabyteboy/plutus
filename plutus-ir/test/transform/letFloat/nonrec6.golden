[
  (lam
    x
    (all a (type) (fun a a))
    (let
      (nonrec)
      (termbind
        (strict)
        (vardecl j (con integer))
        [ [ (builtin addInteger) (con 1) ] x ]
      )
      (termbind (strict) (vardecl i (con integer)) j)
      i
    )
  )
  (con 1)
]